pkg/validations/commandvalidations/validateprovider.go

```diff
@@ -0,0 +1,21 @@
+package commandvalidations
+
+import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/features"
+	"github.com/aws/eks-anywhere/pkg/providers"
+)
+
+func ValidateSupportedProvider(provider providers.Provider) error {
+	if !features.IsActive(features.CloudStackProvider()) && provider.Name() == constants.CloudStackProviderName {
```

Comment:
just curious, where were we running this before?

Reply:
Yes, this func just wraps up the checks we were doing individually in create cluster.

Reply:
Ahhh we do that in the cmd package 🤦🏻‍♂️
Ok, another thing that we should get our of there, eventually